templates/routing-legacy/main.go

```diff
@@ -0,0 +1,59 @@
+// package main holds the implementation of the nextroute template.
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/nextmv-io/sdk/nextroute"
+	"github.com/nextmv-io/sdk/nextroute/factory"
+	"github.com/nextmv-io/sdk/nextroute/schema"
+	"github.com/nextmv-io/sdk/run"
+	runSchema "github.com/nextmv-io/sdk/run/schema"
+)
+
+func main() {
+	runner := run.CLI(solver)
+	err := runner.Run(context.Background())
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+type options struct {
+	Model  factory.Options                `json:"model,omitempty"`
+	Solve  nextroute.ParallelSolveOptions `json:"solve,omitempty"`
+	Format nextroute.FormatOptions        `json:"format,omitempty"`
```

Comment:
How are we going to handle the legacy options? Because they are different.

Reply:
yes, because they are different. @larsbeck and I decided to leave them be.

Reply:
You can't just simply let them be, can you? Then it simply won't work for a customer transitioning that is setting `solver.limits.duration`....

Reply:
yep, I guess we were too focussed on the other use case that Lars has which is only converting old files. So, I will add duration

Reply:
lies, I guess the converter will not handle it but we will set the duration in the app directly, if given.

Reply:
Yeah, good catch. We need to handle the [legacy fleet input options](https://github.com/nextmv-io/cloud/blob/feature/eng-3392-implement-actions-from-on-call/wrangle/jobs/fleet/input.go#L138-L141).
However, we shouldn't be worried about the CLI options, since this is meant for users coming from legacy fleet (which is solely SAS).

Reply:
BTW, not sure what we are planning to do about the [header options](https://github.com/nextmv-io/cloud/blob/feature/eng-3392-implement-actions-from-on-call/core/pkg/data_plane/inputs/fleet.go#L80). Are we supporting them officially?

Reply:
These options are compiled into the binary. That binary is uploaded to the marketplace app and it is what the app uses to run. So that's why I think we definitely have to handle options here. Either here or inside the `solver` func.

Reply:
they are handled in the template that will be the app. to be more precise: only the run duration is handled, because diagram width etc is void and we cannot convert them as far as I know. they are not handled in the input itself because in nextroute they are not part of the input anymore put separate options. or am I missing something?

Reply:
We can deactivate the validation feature, by passing it as `nil`:

```
	runner := run.CLI(solver,
		run.InputValidate[run.CLIRunnerConfig, schema.Input, options, runSchema.Output](
			validate.JSON[schema.Input](nil),
		),
	)
```

Reply:
we talked about this code today and found that it should look like this:

```
runner := run.CLI(solver,
		run.InputValidate[run.CLIRunnerConfig, schema.FleetInput, options, runSchema.Output](
			nil,
		),
	)
```